### ISSUE TITLE
Dns over https ad blocker

### ISSUE THREAD

**Issue by trimechee** [Eloston/ungoogled-chromium#1941]
Hello, it will more easy for newbies if our beloved Ungoogled-Chromium Browser have dns preset value like and include ad blocker dns by default like controld dns, bancuh dns, adguard dns, radical dns, opennic dns.....beacause it may be difficult to remember the value of the dns and change them in dns overt https section, thank you :)

**Comment by PF4Public** [comment]
Are you kidding?
Duplicate of #1939

**Comment by trimechee** [comment]
My suggestion is useful beaucaise Ungoogled -Chromium has disabled standard google protection browsing : I just point out that chromium has a "secure dns" section where there is a default list of secure dns like the one from couldflare, google dns, cleanbrowsing, open dsn, this is a good list that block viruses but don't block ads, d 'where my suggestion to add another ad blocker dns for simplicity, because sometimes the dns become heavy or have false posts and block certain legitimate sites so you can change between the different dns with a single click, for example this dns says it blocks crypto currency mining sites and malware, so with these awiesome dns we no longer need to use anti virus which can weigh down the computer, extensions that block ads are good but the dns blocks ads and malware at source i.e. ads are not loaded at all whereas for ad blocker extensions, ads are loaded by the system and blocked by the adblock extension.....

for exemple this rare dns blocks in the source the ads, the crypto currency minig and manu malware websites.....

Ultimate.Hosts.Blacklist
https://github.com/Ultimate-Hosts-Blacklist/Ultimate.Hosts.Blacklist/issues/590

and OpenNIC

https://servers.opennic.org/

**Comment by PF4Public** [comment]
This could be easily added locally while building from source, couldn't it?

**Comment by Ph0rk0z** [comment]
Your browser has built in DNS? I use dnscrypt which can auto update the blacklist through a script. Please don't put ANY dns in the browser. Even DOH is meh and I'm glad its off by default.

**Comment by trimechee** [comment]
Thank you for advice :)
